lte/gateway/c/oai/common/itti_free_defined_msg.c

```diff
@@ -203,7 +213,9 @@ void itti_free_msg_content(MessageDef* const message_p) {
     case SCTP_CLOSE_ASSOCIATION:
       // DO nothing
       break;
-
+    case TIMER_HAS_EXPIRED:
+       //free_wrapper (&arg);
```

Comment:
```suggestion
       // Do nothing;
```

Reply:
This was done to fix s11 memory leaks and has been updated to an appropriate code.